.travis.yml

```diff
@@ -9,8 +9,10 @@ python:
 
 stages:
   - test-integrade
-  - setup-cloudigrade
-  - test-cloudigrade
+  - name: setup-cloudigrade
+    if: branch = master or (branch != master and type = pull_request)
```

Comment:
I am not sure if we should make it explicit to only test master on push. I think we will never have master serving as PR